cfy_manager/components/postgresql_server/scripts/syslog_wrapper_script.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+rm -rf /var/run/syslogd.pid
```

Comment:
Probably remove the -r from this, as it's just deleting a file. If that path somehow became a directory then failure would be appropriate.

Reply:
sure